srcpkgs/skopeo/template

```diff
@@ -14,16 +15,26 @@ maintainer="Cameron Nemo <cnemo@tutanota.com>"
 license="Apache-2.0"
 homepage="https://github.com/containers/skopeo"
 distfiles="https://github.com/containers/${pkgname}/archive/v${version}.tar.gz"
-checksum=453bdcce16767696ed71046b60ad7b34358b183b50eb5aa708ced0b5ea2927b1
+checksum=9a321ba75f213e5c46cba7f92073c2437137a56d3140c9ab6e723fb92890f9d0
 make_dirs="/var/lib/atomic/sigstore 0755 root root"
+_completions="bash zsh fish"
 
 post_build() {
 	make docs
+	local cli=$(find $GOPATH/bin -name skopeo)
```

Comment:
isn't this the same path every time?

Reply:
I got this from the k3d template, but it works fine if I just replace `$cli` with `$GOPATH/bin/skopeo`.

Reply:
alright I see why k3d uses find... differs based on architecture/native/cross.